src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/CodeDirectiveFormattingTest.cs

```diff
@@ -451,10 +451,7 @@ public class Foo
             {
                 public Foo()
                 {
-                    var arr = new string[ ] {
-""One"", ""two"",
```

Comment:
Lol, what strange formatting we were expecting in the first place.

Reply:
In general, I love the "here is a bunch of code!" complex tests, because some of the other tests are so targetted that its possible to not be valuable, but then you get cases like there where the test covers 5 different things, so when one of them changes its a pain.